libs/ui-lib/lib/ocm/components/featureSupportLevels/featureStateUtils.ts

```diff
@@ -181,7 +199,11 @@ export const getNewFeatureDisabledReason = (
     }
     case 'MCE': {
       if (!isSupported) {
-        return 'Multicluster engine is not supported in this OpenShift version.';
+        if (platformType === 'nutanix') {
```

Comment:
Same question for MCE, if Nutanix is incompatible, we shouldn't check whether it's supported based on other criteria.

Reply:
True, I put the condition outside isSupported check.